server/createEnrollmentsFile.js

```diff
@@ -117,9 +126,20 @@ async function writeUsersForCourse ({canvasCourse, ldapClient, fileName}) {
   for (let user of examinators) {
     await csvFile.writeLine([canvasCourse.sisCourseId, user.ugKthid, 'Examiner', 'active'], fileName)
   }
+
+  // Registered students
+  try {
+    const registeredMembers = await searchGroup(`(&(objectClass=group)(CN=ladok2.kurser.${courseInitials}.${courseCodeWOInitials}.registrerade_${canvasCourse.startTerm}.${roundId}))`, ldapClient)
+    const registeredStudents = await getUsersForMembers(registeredMembers, ldapClient)
+    for (let user of registeredStudents) {
+      await csvFile.writeLine([canvasCourse.sisCourseId, user.ugKthid, 'Student', 'active'], fileName)
+    }
+  } catch (err) {
+    logger.info('Could not get registered students for this course. Perhaps there are no students?', err)
```

Comment:
According to [Bunyan API docs](https://github.com/trentm/node-bunyan#log-method-api), the way of logging error is `logger.info(err)` OR `logger.info(err, msg)` OR `logger.info({err: error}, msg)`